### ISSUE TITLE
[Need QA] Using 'Email' for authentication not working

### ISSUE THREAD

**Issue by mzico** [GluuFederation/QA#4]
Community reported that using 'Email' as 'Primary username' in basic authentication not working. 

[This](https://gluu.org/docs/gluu-server/4.1/authn-guide/basic/#configuring-basic-authentication) is what we need to test in 4.1 and 4.2

**Comment by MohitMaliFtechiz** [comment]
Yes with mail as username its not authenticating. I am not sure its bug because we are asking to enter username not mail. what you thought @mzico

**Comment by mzico** [comment]
@MohitMaliFtechiz : It's bug. 'Username' is actually the 'Primary Key'; you can put UID or EmailAddress there.  Let's open an issue in oxAuth and assign @yurem please.
